ytfzf

```diff
@@ -967,51 +1018,74 @@ user_selection () {
 		export YTFZF_THUMB_DISP_METHOD="$thumb_disp_method"
 		[ "$thumb_disp_method" = "ueberzug" ] && start_ueberzug
 		#thumbnails only work in fzf, use fzf
-		menu_command="fzf -m --tabstop=1 --bind change:top --delimiter=\"$tab_space\" \
-		--nth=1,2 --expect='$shortcuts' $FZF_DEFAULT_OPTS \
-		--layout=reverse --preview \"sh $0 -U {}\" \
+        menu_command="fzf -m --tabstop=1 \
+        --bind 'change:top"$ignore_binds$binds"' --delimiter=\"$tab_space\" \
+		--nth=1,2 --expect='$key_binds' $FZF_DEFAULT_OPTS \
+        --layout=reverse --preview \"sh $0 -U 'preview_img '{}\" \
         	--preview-window \"$PREVIEW_SIDE:50%:noborder:wrap\""
 		selected_data=$( title_len=200 video_menu "$videos_data" )
 		[ "$thumb_disp_method" = "ueberzug" ] && stop_ueberzug
 		# Deletes thumbnails if no video is selected
 		[ -z "$selected_data" ] && clean_up
 	#show regular menu
 	else
+        format_menu
 		selected_data=$( video_menu "$videos_data" )
 	fi
 	unset videos_data_clean
 }
 
-handle_shortcuts () {
+fzf_bind_command () {
+    args=$*
+    selected_data="$( cat "${args#* }" )"
+    format_user_selection
+    selected_key=${*%% *}
```

Comment:
@Euro20179 this isn't POSIX complaint right? store $* in a variable and then trim the variable

Reply:
yes store $* in a variable then use the variable

Reply:
I think I should go back to using `$!`.
@Euro20179 seems to think it does not work in dash, but it works with my dash, so it must be a problem on your end.